pages.de/common/head.md

```diff
@@ -0,0 +1,20 @@
+# head
+
+> Gibt den ersten Teil einer Datei aus.
+> Weitere Informationen: <https://www.gnu.org/software/coreutils/head>.
+
+- Gebe die ersten Zeilen einer Datei aus:
```

Comment:
What do you think about `paar`?
Also, `gebe` is used often in some areas, but not correct.
```suggestion
- Gib die ersten paar Zeilen einer Datei aus:
```